src/log4stash/Filters/JsonFilter.cs

```diff
@@ -76,7 +76,7 @@ private static void ScanToken(IDictionary<string, object> logEvent, JToken token
 
         private static string Join(string prefix, string name)
         {
-            return (string.IsNullOrEmpty(prefix) ? name : prefix + "." + name);
+            return (string.IsNullOrEmpty(prefix) ? name : prefix + "_" + name);

```

Comment:
Can you extract "_" into a property? so it could be configured from outside?
Thanks :)